handbook/engineering/distribution/recurring_processes.md

```diff
@@ -118,33 +129,83 @@ come prepared to talk about the following:
 
 ## Weekly
 
-### Internal sync
+### Company meeting
+
+As with everyone at Sourcegraph, we join the weekly company meeting on Mondays @ 10:30am PST (when timezone permits, watching the recording otherwise).
```

Comment:
```suggestion
As with everyone at Sourcegraph, we join the [weekly company meeting](https://about.sourcegraph.com/handbook/communication/company_meeting) on Mondays @ 10:30am PST (when timezone permits, watching the recording otherwise).
```